Essentials/src/plugin.yml

```
@@ -3,7 +3,7 @@ name: Essentials
 main: com.earth2me.essentials.Essentials
 # Note to developers: This next line cannot change, or the automatic versioning system will break.
 version: ${full.version}
-website: http://tiny.cc/EssentialsCommands
+website: https://essentialsx.net/wiki/Home.html
```

```suggestion
website: https://essentialsx.net/
```

It would be better to link to the homepage, which is what `website` is supposed to link to AFAIK - people can find the docs easily from there.